src/LinearSolvers/ColumnwiseLUSolver.jl

```diff
@@ -0,0 +1,320 @@
+module ColumnwiseLUSolver
+
+export ManyColumnLU, SingleColumnLU
+
+using ..Mesh.Grids
+using ..Mesh.Topologies
+using ..DGmethods
+using ..DGmethods: BalanceLaw, DGModel, VerticalDirection, num_state, num_diffusive
+using ..LinearSolvers
+const LS = LinearSolvers
+using ..MPIStateArrays
+using LinearAlgebra
+using GPUifyLoops
+
+include("ColumnwiseLUSolver_kernels.jl")
+
+abstract type AbstractColumnLUSolver <: AbstractLinearSolver  end
+
+struct ManyColumnLU <: AbstractColumnLUSolver  end
```

Comment:
What is the difference between `ManyColumnLU` vs `SingleColumnLU`?

Reply:
Single column assumes the system is the same for each column.  I have pushed docs that discuss this.